src/tribler/core/utilities/tiny_tribler_service.py

```diff
@@ -20,7 +21,8 @@ class TinyTriblerService:
     All overlays are disabled by default.
     """
 
-    def __init__(self, components: List[Component], timeout_in_sec=None, state_dir=Path('/tmp/tribler')):
+    def __init__(self, components: Optional[List[Component]] = None, timeout_in_sec: Optional[int] = None,
```

Comment:
That was done intentionally because the absence of components is an error and an exception raised here is expected.
But your version is also ok, and I will implement it.

Reply:
Done.